test/infra/infra-mysql56-single/docker-compose-destroy.bash

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+set -o pipefail
+pushd $(dirname $0) &>/dev/null
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)